MetaMask/metamask-mobile
# chore(sentry): don't report `AxiosError: Network Error`

## **Description**

Added a try catch around the infura calls so Sentry does not pick up an error from axois when the request fails due to poor connection.

## **Related issues**

Fixes: [This issue](https://metamask.sentry.io/issues/5665770057/?project=2299799&referrer=github_integration)

## **Manual testing steps**

- Open the app, log in, disconnect from the internet.
- - App should now handle that failure instead of the api throwing an error which is caught by Sentry.

## **Screenshots/Recordings**

### **Before**

<img width="489" alt="Screenshot 2024-10-15 at 14 53 11" src="https://github.com/user-attachments/assets/2e48befa-9e7f-41f2-98e9-e1d3a51cef8d">

### **After**

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
